src/style.css

```diff
@@ -1 +1,89 @@
-/* Styles go here */
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  --background-color: #e5e5e5;
+  --header-color: #fff;
+  --header-height: 60px;
+  --logo-height: 40px;
+  --logo-width: 40px;
+  --link-color: #00acdc;
+}
+
+.body {
+  margin: 0;
+  background-color: var(--background-color);
+}
+
+.header {
+  display: flex;
+  position: relative;
+  align-items: center;
+  justify-content: space-between;
+  background-color: var(--header-color);
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.nav {
+  display: flex;
+  align-items: center;
+  margin-right: 50px;
+}
+
+.logo {
+  padding: 10px 0 10px 50px;
+}
+
+.logo__img {
+  display: block;
+  height: var(--logo-height);
+  width: var(--logo-width);
+}
+
+.nav__list {
+  display: flex;
+  margin: 0;
+  padding: 0;
+  list-style: none;
+  text-transform: uppercase;
+}
+
+.nav__link {
+  display: inline-block;
+  margin-left: 20px;
```

Comment:
You should avoid margin collapsing, when a child margins go through a parent and push off another parent
[useful_link](https://www.smashingmagazine.com/2019/07/margins-in-css/#parent-and-first-or-last-child-element)
So, if you'd like one element to be push off from another element use margins at THESE elements, not their children

P.S. and the first element shouldn't have margin-left. Use `:not(:first-child)`